tests/Integration/Adapter/ContainerBuilderTest.php

```diff
@@ -52,21 +51,6 @@ public function testFrontContainerContainsAnEntityManager()
         $this->assertInstanceOf(EntityManagerInterface::class, $entityManager);
     }
 
-    public function testGetAdminContainer()
```

Comment:
You could add a test here asserting that the `ServiceContainerException` is thrown when trying to access the AdminContainer

Reply:
Done!